nerfstudio/engine/schedulers.py

```diff
@@ -25,6 +25,14 @@
 
 from nerfstudio.configs.base_config import InstantiateConfig
 
+try:
+    from torch.optim.lr_scheduler import (  # pylint: disable=ungrouped-imports
+        LRScheduler,
+    )
+except ImportError:
+    # Backward compatibility for PyTorch 1.x
```

Comment:
This offers backward compatibility for PyTorch 1.x. Should we keep it here or drop it?

Reply:
Lets keep it for now, but print a warning to the user that they should update to pytorch 2.0